File: 22/C#/Day1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Advent22
{
    internal class Day1
    {
        public Day1()
        {
            List<string> File = System.IO.File.ReadLines("C:\\Users\\alexf\\OneDrive\\Documents\\VisualCode\\AdventofCode\\22\\Day1.txt").ToList();

            List<int> Solution = new();
            int run_total = 0;
            foreach (var file in File)
            {
                if (file.Length == 0)
                {
                    Solution.Add(run_total);
                    run_total = 0;
                }
                else
                {
                    run_total += Int32.Parse(file);
                }
            }
            Solution.Add(run_total);
            Solution.Sort();
            Solution.RemoveRange(0, (Solution.Count - 3));
            Console.WriteLine(Solution.Last());
            Console.WriteLine(Solution.Sum());
        }
    }
}

File: 22/Day1.py
Advent = open("C:\\Users\\alexf\\OneDrive\\Documents\\AdventofCode\\22\\Day1.txt", "r")
Advent = Advent.readlines()
Part1 = []
Score = 0
for line in Advent:
        if not line.isspace():
            val = int(line)
            Score += val
        else:
            Part1.append(Score)
            Score = 0
Part1.append(Score)
Part1.sort(reverse = True)
print(Part1[0])
print((Part1[0] + Part1[1] + Part1[2]))